yt/visualization/base_plot_types.py

```diff
@@ -337,8 +336,7 @@ def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
             self.cb.set_ticks(yticks)
         else:
             self.cb = self.figure.colorbar(self.image, self.cax)
-        for which in ["major", "minor"]:
-            self.cax.tick_params(which=which, axis="y", direction="in")
+        self.cax.tick_params(which="both", axis="y", direction="in")
```

Comment:
I guess the explicit loop was necessary with old versions of matplotlib but it's not needed for the current oldest supported version (2.2.0)
see https://matplotlib.org/2.2.0/api/_as_gen/matplotlib.axes.Axes.tick_params.html?highlight=tick_params#matplotlib.axes.Axes.tick_params

Reply:
OK, makes sense.